docs/reference/release-notes/highlights.asciidoc

```diff
@@ -25,6 +25,19 @@ Other versions:
 
 // tag::notable-highlights[]
 [discrete]
+[[indexing-speed-improvement]]
+===  Indexing speed improvement
+
+{es} 7.10 improves the indexing speed by 5-20%, depending on your workload.
+We've reduced the amount of coordination needed to add entries to the
+<<index-modules-translog,transaction log>>, enabling more concurrency, and
+incresed the transaction log buffer size from `8KB` to `1MB`.
```

Comment:
is this too low level? (as it's not something the users can configure)

Reply:
I think this is still worth highlighting as it's a change users may notice or be interested in.